yew/src/callback.rs

```diff
@@ -4,6 +4,24 @@ use std::cell::RefCell;
 use std::fmt;
 use std::rc::Rc;
 
+/// Defines the event listener as passive.
+/// Yew sets sane defaults depending on the type of the listener.
+/// See [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEvent).
+pub const PASSIVE: u8 = 1;
```

Comment:
Using a raw `u8` seems very unergonomic to me. At the very least this should use the newtype idiom.

Reply:
Okay, agreed.